OutOfSchool/OutOfSchool.DataAccess/Repository/IProviderRepository.cs

```diff
@@ -5,8 +5,8 @@
 
 namespace OutOfSchool.Services.Repository
 {
-    public interface IOrganizationRepository : IEntityRepository<Organization>
+    public interface IProviderRepository : IEntityRepository<Provider>
     {
-        bool IsUnique(Organization entity);
+        bool IsNotUnique(Provider entity);
```

Comment:
the old name 'IsUnique' was better
the rule is to not create negating names
imagine how it will look like '!IsNotUnique' how hard it would be to figure out what is the the result of the operation